dev/io.openliberty.org.apache.myfaces.4.0_fat/bnd.bnd

```diff
@@ -39,4 +41,8 @@ javac.target: 11
 	io.openliberty.org.apache.commons.codec;version=latest,\
 	io.openliberty.org.apache.xercesImpl;version=latest,\
 	io.openliberty.jakarta.servlet.6.0;version=latest,\
-	io.openliberty.jakarta.annotation.2.1;version=latest
+	io.openliberty.jakarta.annotation.2.1;version=latest,\
+	xml-apis:xml-apis;version='1.4.01',\
+	com.ibm.websphere.javaee.el.3.0,\
```

Comment:
We should be using the latest Jakarta Expression Language 5.0 here.

Reply:
Removed 'com.ibm.websphere.javaee.el.3.0,\' as it was unnecessary. jakarta EL 5.0 is added below.